### ISSUE TITLE
Implement WinAppSDK scene graph visuals for core React Native host components

### ISSUE THREAD

**Issue by jonthysell** [microsoft/react-native-windows#12989]
The WinAppSDK scene graph (composition) visuals and mapping code for each of the core RN Host Components, aka. the “native UI primitives” that RN expects every native platform to implement. These components are `View`, `Text`, `Image`, `TextInput`, `ScrollView`, `Modal`, `ActivityIndicator`, `Switch`, and `RefreshControl`. Without these core components, RN app developers cannot implement even the simplest of RN experiences.

**Comment by chrisglein** [comment]
> [@jonthysell](https://github.com/jonthysell) [@chrisglein](https://github.com/chrisglein) Is this not already done because we have implemented the controls?

@chiaramooney @jonthysell Agreed. I'm not sure what this is tracking at this point. No sub issues, so not being used as a parent for the component tracking. And outside of RefreshControl, this work is already done (moved to individual per-component gap issues). I'm resolving this completed.
